tema27112019/src/main/java/ex4_Produs/Carne.java

```diff
@@ -0,0 +1,20 @@
+package ex4_Produs;
+
+public class Carne extends Produs {
+
+    private TipCarne tipcarne;
+
+
+    public Carne(double price, String nume, String productor, UnitateDeMasura unitateDeMasura, TipCarne tipcarne) {
```

Comment:
Carnea in general este la kg. (Nu vreau ca cineva sa aiba posibilitatea de a crea carne cu unitatea de masura bucata).
Sterge parametrul  UnitateDeMasura unitateDeMasura, din constructor ,  si apeleaza superconstructorul :
 **super(price, nume, productor, UnitateDeMasura.Kilogram);**